webrtc/videoroom/config/ice.exs

```diff
@@ -0,0 +1,6 @@
+import Config
```

Comment:
I am not sure if this should be a separate file. Now we have so many config files

Reply:
I'm pretty sure we should not add config files that are not default to elixir ones (I mean the names). If this is hard coded then it should probably be put in `config/config.exs` but I guess we would like to provide stun servers in runtime.

Reply:
But I have no idea what should be the format for turn servers as they require username and password.

Reply:
why can't we just put that in config.exs or runtime.exs?